### DISCUSSION TITLE
Use OpenTrack With Virtual Reality Headset In Mechwarrior 5

### DISCUSSION THREAD

**Discussion by JazzyJazzjazzy** [opentrack/opentrack#1551] [Q&A]
Hello,

Thanks for the great software. How can I implement OpenTrack in Mechwarrior 5 VR game to respond to VR headset movement? It is a Steam Game. Thank you.

![1](https://user-images.githubusercontent.com/40358464/196014404-fef12695-ab40-46f4-9d24-62ee7c519553.jpg)

**Comment by Slion** [comment]
Last I checked MW5 did not support TrackIR. I would be interested if you find a way to get this working somehow.

**Reply by JazzyJazzjazzy** [reply]
Hello SLION,

Thank you for your kind reply. Natively, MW5 does NOT support any sort of tracking which is why I am here. As a side note, MW5 does not natively support VR either, but I run it in VR everyday.

mods.com/mechwarrior5mercenaries/mods/531

I am hopeful that I can receive enough information here to move forward. 

"A quick googling shows something like this Steam Guide https://steamcommunity.com/sharedfiles/filedetails/?id=593645539or AVSIM https://www.avsim.com/forums/topic/497411-workaround-trackir-chaseplane-vjoy-freepie/

The key here seems to be FreePIE https://andersmalmgren.github.io/FreePIE/ , the posts are old and there may be better ways."

Thank you.

**Reply by JazzyJazzjazzy** [reply]
The ABOVE references did not help. I got it to work using the following...

OpenTrack Settings

Input: SteamVR
Output: Mouse Emulation

Edit Mouse Settings Button>Select X AXIS-YAW, Y AXIS-PITCH

Method: Direct Input

Now, if you can tell me how to separate the Targeting Reticule from the Direction Reticule? I want to control only the Targeting Reticule?

**Reply by Slion** [reply]
Sounds like an issue with MW5 itself rather than OpenTrack.

**Reply by sthalik** [reply]
Head tracking can work only if a game actually supports it.
